shared/chat/conversation/messages/cards/team-journey/container.tsx

```diff
@@ -212,7 +213,11 @@ const TeamJourneyConnected = Container.connect(
       dispatch(Chat2Gen.createPreviewConversation({channelname, reason: 'journeyCardPopular', teamname})),
     _onManageChannels: (teamID: string) =>
       dispatch(
-        RouteTreeGen.createNavigateAppend({path: [{props: {teamID}, selected: 'chatManageChannels'}]})
+        RouteTreeGen.createNavigateAppend({
```

Comment:
you could avoid copying this all over with a new action or making a helper that can be removed later